docs/dir_structure.md

```diff
@@ -382,39 +382,124 @@ mintpy.load.waterMaskFile    = $DATA_DIR/SanFranSenDT42/mask/watermask.msk
+│   ├── S1AA_20160112T205113_20160229T205112_VVP048_INT80_G_ueF_4446_unw_phase_clip.tif
+│   ├── S1AA_20160112T205113_20160229T205112_VVP048_INT80_G_ueF_4446_unw_phase_clip.tif.rsc
+│   ├── S1AA_20160112T205113_20160229T205112_VVP048_INT80_G_ueF_4446_unw_phase.kmz
+│   ├── S1AA_20160112T205113_20160229T205112_VVP048_INT80_G_ueF_4446_unw_phase.png
+│   ├── S1AA_20160112T205113_20160229T205112_VVP048_INT80_G_ueF_4446_unw_phase.png.aux.xml
+│   ├── S1AA_20160112T205113_20160229T205112_VVP048_INT80_G_ueF_4446_unw_phase.tif
+│   ├── S1AA_20160112T205113_20160229T205112_VVP048_INT80_G_ueF_4446_vert_disp_clip.tif
+│   └── S1AA_20160112T205113_20160229T205112_VVP048_INT80_G_ueF_4446_vert_disp.tif
+├── S1AA_20160112T205113_20160229T205112_VVP048_INT80_G_ueF_4446.zip
+
+.
+.
+.
+
+├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2_amp_clip.tif
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2_amp.tif
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2_color_phase.kmz
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2_color_phase.png
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2_color_phase.png.aux.xml
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2_corr_clip.tif
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2_corr_clip.tif.rsc
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2_corr.tif
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2_dem_clip.tif
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2_dem.tif
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2_inc_map_clip.tif
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2_inc_map_clip.tif.rsc
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2_inc_map.tif
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2.README.md.txt
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2.txt
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2_unw_phase_clip.tif
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2_unw_phase_clip.tif.rsc
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2_unw_phase.kmz
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2_unw_phase.png
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2_unw_phase.png.aux.xml
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2_unw_phase.tif
+│   ├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2_vert_disp_clip.tif
+│   └── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2_vert_disp.tif
+├── S1AA_20160112T205113_20160628T205108_VVP168_INT80_G_ueF_73B2.zip
 
-The corresponding template options for `load_data`:
+```
```

Comment:
I don't think we need to change the example file structure here, because:
1. I removed the complete list of files on purpose during previous PRs, so that the list is not too long, and I still think shorter is better.
2. having the DEM and incidence angle files insides one of the pairs is fine, as long as there is at least one. mintpy will search and find it.